packages/components/src/components/alert/alert.component.ts

```diff
@@ -78,23 +82,61 @@ export default class SynAlert extends SynergyElement {
    */
   @property({ type: Number }) duration = Infinity;
 
+  /**
+   * Enables a countdown that indicates the remaining time the alert will be displayed.
+   * Typically used to indicate the remaining time before a whole app refresh.
+   */
+  @property({ type: String, reflect: true }) countdown?: 'rtl' | 'ltr';
```

Comment:
Have you discussed this `countdown` property (and especially its appearance) with Stefan? 
I think it looks a bit strange with our design of the alert (the highlight line on the left side instead of the top like shoelace). Especially the pixel in the left bottom, which is just grey)

![alert-countdown](https://github.com/user-attachments/assets/a4fd3785-574c-4d21-8c53-1a88e58f4131)

Reply:
As discussed we will vendor this away to make the branch releasable